src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="rus">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    data-qa="hover">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" 
+      alt="imac" 
+      class="card__img"
+      >
+      <h1 
+      class="card__textmodel">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h1>
+      <h2 
+      class="card__productcode">Код товара: 195434
+      </h2>
+      <div 
+      class="card__feedback">
+        <div 
+        class="stars">
+          <img src="./images/star-active.svg" 
+          alt="star" 
+          class="star star--staractive">
```

Comment:
```suggestion
          class="star stars__star">
```
it should be an element of the `stars` block
Amount of active stars should be defined in stars block with modificators like the following: `stars--4`